dp-cantabular-csv-exporter.nomad

```diff
@@ -69,7 +69,70 @@ job "dp-cantabular-csv-exporter" {
       }
 
       vault {
-        policies = ["dp-cantabular-csv-exporter"]
+        policies = ["dp-cantabular-csv-exporter-publishing"]
+      }
+    }
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "publishing"
```

Comment:
This value should be "web"

Reply:
Thank you @sandypadmanabhan!
I have addressed your comment :)